lib/models/movie.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
+    return {
+      id: this.get('id'),
+      title: this.get('title'),
+      release_year: this.get('release_year'),
+      object: 'movie'
+    };
+  }
+});
```

Comment:
looks like you're missing a newline at the end of the file. some editors treat this differently, but it's common to end all text-like files with a `\n`. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline